### DISCUSSION TITLE
Community Meeting - 2024‐11‐21 08:00 UTC

### DISCUSSION THREAD

**Discussion by Firstyear** [kanidm/kanidm#3181] [General]
A community meeting will be held at 2024‐11‐21 08:00 UTC

The agenda can be found here:

https://github.com/kanidm/kanidm/wiki/Community-Meeting-%E2%80%90-2024%E2%80%9011%E2%80%9014-08:00-UTC

The meeting will be held on [jitsi](https://meet.jit.si/KanidmCommunityMeeting)

**Comment by ToxicMushroom** [comment]
I've had a bad experience with the public jitsi instance (quite laggy voice communication, voice would be delayed and then sped up to catch up resulting in unnatural communication). Though this was last year maybe it is better now.
